service/history/shard/controller_impl.go

```diff
@@ -88,6 +89,8 @@ type (
 		archivalMetadata            archiver.ArchivalMetadata
 		hostInfoProvider            membership.HostInfoProvider
 		tracer                      trace.Tracer
+		acquireShardRetryPolicy     backoff.RetryPolicy
```

Comment:
I don't think we need it here. We can create the policy within shard context, and test can directly update that field to a test version.

Reply:
Moved